packages/donor/src/state/AppointmentToBookStore.ts

```diff
@@ -33,7 +33,7 @@ export class AppointmentToBookStore {
     this.appointmentIds = slot.appointmentIds;
   }
 
-  isDonationValidForDonor(donor: Donor): boolean {
+  isAppointmentTooCloseToLastDonation(donor: Donor): boolean {
```

Comment:
Not really sure how to do this (and whether this is hard or not) but this seems like a function that could use a test.

If you have the time, please try to add a test to this one